README.md

```diff
@@ -229,6 +229,26 @@ If you are using the open source terraform-aws-vpc module, you can set `nat_gate
 
 AlterNATively, you can remove the NAT Gateways and their EIPs from your existing configuration and then `terraform import` them to allow alterNAT to manage them.
 
+#### Providing explicit Elastic IPs for fallback NAT Gateways
+
+Starting with this version, you can optionally supply your own Elastic IP allocation IDs for **fallback NAT Gateways** instead of letting alterNAT create them automatically.
```

Comment:
```suggestion
You can optionally supply your own Elastic IP allocation IDs for the fallback NAT Gateways instead of letting alterNAT create them automatically.
```